tests/utils.go

```diff
@@ -3070,6 +3126,9 @@ func RenderPod(name string, cmd []string, args []string) *k8sv1.Pod {
 		ObjectMeta: metav1.ObjectMeta{
 			GenerateName: name,
 			Namespace:    NamespaceTestDefault,
+			Annotations: map[string]string{
+				IstioInjectSidecarAnnotation: "false",
+			},
```

Comment:
I'm not sure how I feel about this. Why would all pods we use have to have this ?

Reply:
I tried to address this in [this commit](https://github.com/kubevirt/kubevirt/pull/5200/commits/2667f5c2120ef2411079d0f3d239e2b0b2d418c1).
A simple alternative would be removing the label just before creating the Job, but that felt a little bit ackward.